protocols/meson.build

```diff
@@ -1,22 +1,25 @@
+wl_protocol_dir = wayland_protos.get_variable('pkgdatadir')
+
 wayland_scanner_dep = dependency('wayland-scanner', required: false, native: true)
 if wayland_scanner_dep.found()
 	wayland_scanner = find_program(
-		wayland_scanner_dep.get_pkgconfig_variable('wayland_scanner'),
+		wayland_scanner_dep.get_variable(pkgconfig: 'wayland_scanner'),
 		native: true,
 	)
 else
 	wayland_scanner = find_program('wayland-scanner', native: true)
 endif
 
 client_protocols = [
-	'linux-dmabuf-unstable-v1.xml',
-	'wlr-screencopy-unstable-v1.xml',
-	'xdg-output-unstable-v1.xml',
+	[wl_protocol_dir, 'unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml'],
```

Comment:
Nit: can be simplified with `/`: `wl_protocol_dir / 'unstable/linux-dmabuf/…'`

Reply:
done